tools/travis/travis_linux_script.sh

```diff
@@ -233,28 +233,50 @@ else
         echo -en 'travis_fold:end:script.build.multithread_discovery\\r'
     fi
 
-    echo -e "\r\n== Unit tests (full NS0) ==" && echo -en 'travis_fold:start:script.build.unit_test_ns0_full\\r'
+    #echo -e "\r\n== Unit tests (full NS0; Debug) ==" && echo -en 'travis_fold:start:script.build.unit_test_ns0_full_debug\\r'
```

Comment:
Is there a specific reason why you remove the full NS0 Debug build?

Reply:
I thought that a Debug build is basically a subset of the Release build.
So if the unit tests run with the Release, they almost sure run with the Debug build and therefore we can save time.

Reply:
Nope :)
Within the stack we use in some places an `UA_assert`, e.g. here:
https://github.com/open62541/open62541/blob/74b7578cd7e9569beb5bb64098a0dd8ead5cf846/plugins/ua_nodestore_default.c#L280

This assert for example checks, if all the node references are correctly released. `UA_assert` is only enabled on debug builds, to keep the performance impact low, but still show an error during developing.
Therefore it would make sense to build open62541 with all the features enabled and run all the unit tests, to also cover these asserts.

Reply:
Thanks, didn't know that! :)
Ok so for now it might be a good idea to run the full NS0 just in debug mode in the CI build routine.

I'm currently thinking about a couple of flags pre-sets for different build and optimization types. Additionally, they should not be set globally but as target properties.